### ISSUE TITLE
Facet Re-design for Simple and Advanced Search

### ISSUE THREAD

**Issue by roamye** [project-lux/lux-frontend#416]
**Problem Description:**
LUX would like to redesign the front end layout based on POW's mockups. This issue will support the re-design of the front end layout (#270) and specifically focus on the facets on both simple and advanced search. 

**This issue needs to be done before the tab re-designs for simple and advanced search**

**Expected Behavior/Solution:**
LUX facets (advanced and simple search) to be re-configured to match the requirements and mockups below. 

**Requirements:**
For both Advanced Search and Simple Search. 
- Merge `X <scope> results` block with `facet` block. 
 - - the Title, description and toggles of the `X <scope> result` block will remain at the top. 
- - facet and results will be displayed underneath the `X <scope> results description` and toggles. 
- - a blue line will separate the description & search toggles from the merged facet and results block. 
- - facets will remain on the left hand side
- - search results will remain on the right hand side 
- - a grey separate will be added to define the facet block from the search result block. 
- Facets label `Refine <scope>` will be removed. 
- help icon `?` will be added to the right-hand side of the title. 

**Needed for promotion:**
If an item on the list is not needed, it should be crossed off but not removed.

- [ ] Wireframe/Mockup - Mike
- [ ] Committee discussions - Sarah
- [ ] Feasibility/Team discussion - Sarah
- [ ] Backend requirements - TBD
- [ ] Frontend requirements- TBD
- [ ] Are new regression tests required for QA - Amy
- [ ] Questions
- List of questions for discussions. Answers should be documented within the issue.

**UAT/LUX Examples:**
- https://lux-front-tst.collections.yale.edu/view/results/objects?q=%7B%22AND%22%3A%5B%7B%22text%22%3A%22andy%22%2C%22_lang%22%3A%22en%22%7D%2C%7B%22text%22%3A%22warhol%22%2C%22_lang%22%3A%22en%22%7D%5D%7D&sq=andy+warhol

**Dependencies/Blocks:**
- Blocking: Issues being blocked by the completion of the current issue.

**Related Github Issues:**
- META: https://github.com/project-lux/lux-frontend/issues/270

**Related links:**
[Presentation](https://www.figma.com/proto/bWS2orxFECNrG92HfG48BL/LUX-Yale-Design-Walkthrough?page-id=3377%3A9485&node-id=3377-9572&viewport=4999%2C493%2C0.26&t=Axk0QSGESY3qeZh1-1&scaling=scale-down&content-scaling=fixed)
[Design walkthrough](https://www.figma.com/design/bWS2orxFECNrG92HfG48BL/LUX-Yale-Design-Walkthrough?node-id=30-6364&node-type=canvas&t=Sy4U7OxDEOrthtNY-0)
[Prototype](https://www.figma.com/proto/bWS2orxFECNrG92HfG48BL/LUX-Yale-Design-Walkthrough?page-id=182%3A6364&node-id=3317-20738&viewport=-5411%2C-5010%2C0.36&t=yi4XGHaZfpsKRsqT-9&scaling=min-zoom&content-scaling=fixed&starting-point-node-id=3317%3A20738&show-proto-sidebar=1)

**Wireframe/Mockup:**
Current: 
Simple Search
![image](https://github.com/user-attachments/assets/612354f9-1c36-4fab-b642-bdaf07e71f4d)
Advanced Search 
![image](https://github.com/user-attachments/assets/222cd00d-9611-42ea-b467-21467d8f6c14)

Expected: 
Simple Search 
![image](https://github.com/user-attachments/assets/d3c041e5-5527-45f1-9ebb-74b32047440e)
Advanced Search 
![image](https://github.com/user-attachments/assets/c4de7f33-581e-4173-82a3-087ace4e0d1f)

**Comment by roamye** [comment]
Looks good in PROD. 
New ticket no longer needed as (?) icon has been added. 
Closing.
